exam_1/task1/main.c

```diff
@@ -0,0 +1,99 @@
+#include <stdio.h>
+#include <string.h>
+
+int tripleTransformation(char *binary, char *octale)
+{
+    int result = 0;
```

Comment:
```suggestion
    char transformedTruad = 0;
```
1) Должно быть более информативное имя. (-0.25)
2) Тип должен быть char. (-0.25)